roles/openshift_health_checker/openshift_checks/etcd_volume.py

```diff
@@ -0,0 +1,54 @@
+# pylint: disable=missing-docstring
```

Comment:
[PyLint rules MUST NOT be disabled on a whole file.](https://github.com/openshift/openshift-ansible/blob/master/docs/best_practices_guide.adoc#pylint)

And I suppose I'd rather have the docstrings, or make the methods internal, than have this everywhere.

[Also from the same doc](https://github.com/openshift/openshift-ansible/blob/master/docs/best_practices_guide.adoc#python-source-files):

    # vim: expandtab:tabstop=4:shiftwidth=4